templates/generic_cell_clustering.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>It's better to allow the user to set <code>normalize</code> in section 1. Either put it in its own cell or add it to one of the other param setting cells.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1027/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>